mkosi/__init__.py

```diff
@@ -3665,7 +3665,15 @@ def insert_partition(
 
             # Without this the entire blob will be read into memory which could exceed system memory
             with open(path, mode='wb') as path_fp:
-                os.sendfile(path_fp.fileno(), blob.fileno(), offset=0, count=blob_size)
+                # Chunk size for 32/64-bit systems
+                # Chunking required because sendfile under Linux has a maximum copy size
+                chunksize = 2 ** 30 if sys.maxsize < 2 ** 32 else 0x7ffff000
```

Comment:
Shouldn't this be `<=`?

Reply:
I copied this from the shutil code [here](https://github.com/python/cpython/blob/main/Lib/shutil.py#L138), but according to the official documentation it should really be `< 2 ** 31` since `sys.maxsize = 2 ** 31 - 1` for 32-bit systems (see [here](https://docs.python.org/3/library/sys.html#sys.maxsize)).